00-programming-fundamentals/installfest.md

```diff
@@ -31,50 +32,54 @@ If the Xcode Command-Line tools are not installed you can install it with:
 If the command does not show a directory path and instead something like `xcode-select: error: unable to get active developer directory, use xcode-select --switch to set one (or see man xcode-select)` you can install it with `$ xcode-select --install` select install and then grab coffee because it will take a bit of time.
 
 ## Homebrew
-[Homebrew](http://brew.sh/) is a **package manager** that simplifies installing software on macOS. Package managers are very common, and each language/framework/stack tends to have their own or preferred tool.
+
+[Homebrew](http://brew.sh/) is a **package manager** that simplifies installing software on macOS. Package managers are very common, and each language/framework/stack tends to have their own or preferred tool.  Homebrew should also have been installed in Jumpstart.
 
 To install, copypasta this into your terminal:
 
 `$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
 
 You may need to enter your computer password to complete the installation.
 
-### Trust but Verify
+### Trust but Verify Brew
+
 Run `$ brew doctor`. Brew is super great at telling you what else it may need. Sometimes there are file permissions to fix. Sometimes there's a warning that can be ignored. If there's anything ever wrong with Brew, `brew doctor` will usually be able to tell you what it is and how to fix it.
```

Comment:
I tend to think that `$`s at the beginnings of lines are confusing inline.  (Though we do this all over the place here so I get if you want to leave it.)

```suggestion
Run `brew doctor`. Brew is super great at telling you what else it may need. Sometimes there are file permissions to fix. Sometimes there's a warning that can be ignored. If there's anything ever wrong with Brew, `brew doctor` will usually be able to tell you what it is and how to fix it.
```